### PR TITLE
docs(schematics): fixing inconsistencies

### PR DESCRIPTION
**Pull request by bougwal**
I fixed inconsistent spelling and formatting and replaced a dead link to clarity design system starter.

## PR Checklist
Please check if your PR fulfills the following requirements:

- [x] The commit message follows our guidelines: https://github.com/angular/angular/blob/main/CONTRIBUTING.md#commit
- [ ] Tests for the changes have been added (for bug fixes / features)
- [x] Docs have been added / updated (for bug fixes / features)

## PR Type
What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature
- [x] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [x] Documentation content changes
- [ ] angular.io application / infrastructure changes
- [ ] Other... Please describe:

## What is the current behavior?

N/A
Issue Number: N/A
N/A

## What is the new behavior?

N/A

## Does this PR introduce a breaking change?

- [ ] Yes
- [x] No

## Other information
N/A

### CODE REVIEW

File: aio/content/guide/schematics.md [hunk 1]
```diff
@@ -16,8 +16,8 @@ You can modify these schematics, and define new ones to do things like update yo
 
 Schematics that are included in the `@schematics/angular` collection are run by default by the commands `ng generate` and `ng add`.
 The package contains named schematics that configure the options that are available to the CLI for `ng generate` sub-commands, such as `ng generate component` and `ng generate service`.
-The subcommands for `ng generate` are shorthand for the corresponding schematic.
-To specify a particular schematic \(or collection of schematics\) to generate, using the long form:
+The sub-commands for `ng generate` are shorthand for the corresponding schematic.
+To specify and generate a particular schematic \(or a collection of schematics\) using the long form:
```
Comment (bob-watson) [inline aio/content/guide/schematics.md:20]: The parentheses aren't necessary
```suggestion
To specify and generate a particular schematic, or a collection of schematics, using the long form:
```

**Review by bob-watson — COMMENTED** [review]
Thanks for the edits.
I just have one suggestion for readability, otherwise, it LGTM.

File: aio/content/guide/schematics.md [hunk 2]
```diff
@@ -46,8 +46,8 @@ The schema describes the options available to the CLI for each of the `ng genera
 
 As a library developer, you can create your own collections of custom schematics to integrate your library with the Angular CLI.
 
-*   An *add schematic* lets developers install your library in an Angular workspace using `ng add`
-*   *Generation schematics* can tell the `ng generate` subcommands how to modify projects, add configurations and scripts, and scaffold artifacts that are defined in your library
+*   An *add schematic* lets developers install their library in an Angular workspace using `ng add`
```
Comment (alan-agius4) [inline aio/content/guide/schematics.md:49]: This change should be reverted.

Comment (walidBouguima) [reply aio/content/guide/schematics.md]: Got it! Thanks for your review @alan-agius4, change reverted.

**Review by bob-watson — CHANGES_REQUESTED** [review]
LGTM

Can you squash the two commits into one, please?
Also, when you do that, the resulting commit message should start with just `docs: `

Thanks!

reviewed-for: global-docs-approvers

**Comment by walidBouguima** [comment]
Done @bob-watson, Thanks for your comment!

**Review by bob-watson — CHANGES_REQUESTED** [review]
Thanks for the edits, and squashing the commits.
Please remove the parentheses as noted, and squash the resulting commits into one commit as you've done here.
After that, it'll be ready to go.
Thanks!

File: aio/content/guide/schematics.md (hunk 1 above)
Comment (walidBouguima) [reply aio/content/guide/schematics.md]: Thanks for your feedback @bob-watson ! The parentheses were already there and not added. Now, they are removed as per your suggestion (which it seems I missed for three weeks). Thank you!

**Review by bob-watson — APPROVED** [review]
LGTM

Reviewed-for: global-docs-approvers

@alan-agius4 can you give it another look and review/approve it for merging, please?
Thanks!

**Review by alan-agius4 — APPROVED** [review]

**Comment by alan-agius4** [comment]
@walidBouguima, can you please rebase?

**Review by alan-agius4 — APPROVED** [review]
LGTM

**Comment by pkozlowski-opensource** [comment]
This PR was merged into the repository by commit 2f32ce071e563b199e543cc10094b08b041986b3.

**Pull request closed without merging**